libraries/cli/include/cli/default_config.hpp

```diff
@@ -16,6 +16,8 @@ const char *default_json = R"foo({
   "network_sync_level_size": 25,
   "network_packets_processing_threads": 14,
   "network_peer_blacklist_timeout" : 600,
+  "network_peer_light_node_timeout" : 600,
+  "light_node_history" : 0,
```

Comment:
This is one of the very few config parameters that users might actually touch. I would suggest to make it more user friendly and have here flag `"is_light_node" : true/false` + `"light_node_history"`, which would be set to some default value and users would need only change is_light_node param

Reply:
Done